virttest/utils_test/__init__.py

```diff
@@ -139,49 +168,64 @@ def update_boot_option(vm, args_removed=None, args_added=None,
     :raise exceptions.TestError: Raised if fail to update guest kernel cmdline.
 
     """
+    session = None
     if vm.params.get("os_type") == 'windows':
         # this function is only for linux, if we need to change
         # windows guest's boot option, we can use a function like:
         # update_win_bootloader(args_removed, args_added, reboot)
         # (this function is not implement.)
         # here we just:
-        return
-
+        msg = "update_boot_option() is supported only for Linux guest"
+        error_context.context(msg, logging.warn)
+        return False
```

Comment:
I think it is not necessary to return a boolean value since the function call would raise an exception if fail to update guest kernel cmdline, otherwise, we should report `True` in the end of the function and also update the docstring.

Reply:
sure, fixed it.